shared/search/providers.ts

```diff
@@ -527,9 +527,9 @@ function sortByProximity(locations: sourcegraph.Location[], currentURI: URL): so
 function jaccardIndex<T>(a: Set<T>, b: Set<T>): number {
     return (
         // Get the size of the intersection
-        new Set([...[...a]].filter(value => b.has(value))).size /
```

Comment:
Now I'm really curios why it was the way it was and what made it go  away? Tanspiling?

Reply:
https://github.com/sourcegraph/code-intel-extensions/pull/384/files#diff-d7d6c42b22b4cc3c9e90d696d64e26eeR530 🤷

Reply:
This is really interesting because while the code before was already double-copying the arrays (not good), the autofixed double spread looks equivalent and not like it shouldn't work or cause the RangeError 🤔

Reply:
This didn't fix the error, I just cleaned this up since I noticed it. I have no clue why it was doing a double copy in the first place.